<launch>

<node pkg = "ins_eskf" type = "ins_eskf_node" name="ins_eskf_node"/>
<node pkg="rosbag" type="play" name="bag_play_node" args="-s 10 /home/yyj/bag_file/zhihu/kitti_lidar_only_2011_10_03_drive_0027_synced.bag"/>

</launch>